cpp/open3d/t/pipelines/registration/Registration.cpp

```diff
@@ -40,16 +40,15 @@ namespace t {
 namespace pipelines {
 namespace registration {
 
-static void GetRegistrationResultAndCorrespondences(
+static RegistrationResult GetRegistrationResultAndCorrespondences(
```

Comment:
- Correspondences are not returned. Rename the function?
- `GetXXX` typically implies a simple getter function. Maybe rename `Get` -> `Compute).

Reply:
Done. `Correspondences` are part of `RegistrationResult` itself, therefore renamed the function to `ComputeRegistrationResult`